ext/cl_khr_command_buffer_mutable_dispatch.asciidoc

```diff
@@ -101,6 +102,9 @@ typedef cl_uint cl_mutable_command_info_khr;
 
 // Identifies the type of a structure to allow structure pointer chains
 typedef cl_uint cl_command_buffer_structure_type_khr;
+
+// Bitfield covering certain promises by the user to the implementation, enabling possible optimizations 
+typedef cl_bitfield cl_mutable_dispatch_promises_khr;
```

Comment:
We need to define this in the XML, for example https://github.com/KhronosGroup/OpenCL-Docs/blob/c37e00f1423379a882c173c12ed1aa5ca9d8e51d/xml/cl.xml#L254

Reply:
It is defined - in line 255.